### ISSUE TITLE
Windows portable edition

### ISSUE THREAD

**Issue by cmd-rm-rf** [JorgenMygindPilgaard/MemoryMate#100]
It's great app.
Can we humbly asking for a portable edition on windows?
Thank you for bring us this great product.

**Comment by JorgenMygindPilgaard** [comment]
That was actually a great idea. I will see what I can do. It should be possible.

**Comment by JorgenMygindPilgaard** [comment]
Yes, well, the zip-file can't be used as a portable edition, I am afraid. 
Now that we go into the period with dark winther-evenings, I will try to prioritise it. I will keep you posted here 😊

**Comment by JorgenMygindPilgaard** [comment]
Hi @cmd-rm-rf 
The distributable version of the program is now available in [release 2.5.0](https://github.com/JorgenMygindPilgaard/MemoryMate/releases/tag/2.5.0)
